packages/@aws-cdk/aws-ecs/test/fargate/integ.firelens-cloudwatch.ts

```diff
@@ -0,0 +1,89 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as elbv2 from '@aws-cdk/aws-elasticloadbalancingv2';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import * as ecs from '../../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-ecs-integ');
+
+const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 2 });
+
+// grant cloudwatch logs permissions to task role
+const taskIAMRole = new iam.Role(stack, 'firelens-cloudwatch-role', {
+  assumedBy: new iam.ServicePrincipal('ecs-tasks.amazonaws.com'),
+  inlinePolicies: {
+    firelens_cloudwatch_policy: new iam.PolicyDocument({
+      statements: [
+        new iam.PolicyStatement({
+          actions: [
+            'logs:CreateLogStream',
+            'logs:CreateLogGroup',
+            'logs:DescribeLogStreams',
+            'logs:PutLogEvents'
+          ],
+          resources: ['*'],
+        })
+      ]
+    }),
+  },
+});
+
+const cluster = new ecs.Cluster(stack, 'FargateCluster', { vpc });
+
+const taskDefinition = new ecs.FargateTaskDefinition(stack, 'TaskDef', {
+  memoryLimitMiB: 1024,
+  cpu: 512,
+  taskRole: taskIAMRole,
+});
+
+// add log router container with firelens configuration
+taskDefinition.addContainer('log-router', {
+  image: ecs.ContainerImage.fromRegistry('amazon/aws-for-fluent-bit'),
```

Comment:
yes, since there's no SSM parameters for fluentbit image URI, I created a region-image table and the image will be selected by Fn::FindInMap (if no region specified in stack), or get image URI directly if region has been specified: https://github.com/aws/aws-cdk/pull/5457/files#diff-2d9d3b0b1534ae5c6ddea41a79e3c74bR159